mlos_bench/mlos_bench/tests/conftest.py

```diff
@@ -6,18 +6,77 @@
 Common fixtures for mock TunableGroups and Environment objects.
 """
 
+from typing import Any, Dict
+
+import json
```

Comment:
json5, maybe? that's what we use to load the configs

Reply:
Done.